### ISSUE TITLE
"symbolspath" not respected in "Full" and "FastLink" configurations

### ISSUE THREAD

**Issue by mzuther** [premake/premake-core#1006]
Hi!

I am using premake 5.0.0 alpha 12 and have found that the `symbolspath` directive is not respected under certain conditions. Here is a minimal script:

```
workspace "test"
    configurations { "Debug", "Release" }
    location (os.target() .. "/" .. _ACTION .. "/")

    filter { "configurations:Debug" }
        symbols "On"
        symbolspath "$(OutDir)$(TargetName)$(TargetExtension).pdb"

    project ("test")
        kind "WindowedApp"
```

This adds the expected lines

̀```
<ProgramDatabaseFile>$(OutDir)$(TargetName)$(TargetExtension).pdb</ProgramDatabaseFile>
̀```

to the Visual Studio project. But when you change `symbols` to either `Full` or `FastLink`, these lines are not created. I have tested this with the command line options `vs2015` and `vs2017` which both behave in the same way.

I most of the time develop in Linux, so I am not sure whether this is an expected behaviour, but I suspect that it might be a bug...

Thanks for your time,

Martin

**Comment by samsinsane** [comment]
This definitely looks like a bug, [this line](https://github.com/premake/premake-core/blob/0eda22c602567e8b2efb9f1ba104fa766d5adbd4/modules/vstudio/vs2010_vcxproj.lua#L2180) indicates to me that it was created when `symbols` was a Boolean API or even back when it was part of the `flags` API.

**Comment by tdesveauxPKFX** [comment]
Fixed by #1030
